### PR TITLE
Handle UnicodeEncodeError on print.

### PR DESCRIPTION
**Pull request by therealmitchconnors**
I think your bot is stuck on some bad unicode.  Here is a patch.

### CODE REVIEW

File: chessbot.py [hunk 1]
```diff
@@ -255,7 +255,7 @@ def addSubmissionToResponses(submission, fen, certainty, side, responses_filenam
             print("> %s - Responding to %s: %s" % (datetime.now(), submission.id, submission))
             
             # Reply with comment
-            submission.add_comment(msg)
+            # submission.add_comment(msg)
```
Comment (apetresc) [inline chessbot.py]: Probably didn't mean to comment this out, right?

**Comment by therealmitchconnors** [comment]
Correct.  Fixed unintentionally commented code.

**Comment by Elucidation** [comment]
Thanks for noticing + fixing the bug, I'll update the bot tonight and if it works merge the code in

**Comment by Elucidation** [comment]
I checked the bot, looks like the server lost internet connection for the last day or so, I'm in the process of moving my bots onto dockerized google cloud instances to solve this problem. So it doesn't look like there were issues with Unicode errors, however this is still a valid bug that could rear it's head later, so merging in.

**Pull request merged**